sdk/lib/opentelemetry/sdk/trace/export/in_memory_span_exporter.rb

```diff
@@ -37,10 +37,10 @@ def initialize
             @mutex = Mutex.new
           end
 
-          # Returns a frozen array of the finished {Span}s, represented by
+          # Returns a frozen array of the finished {SpanData}s, represented by
           # {io.opentelemetry.proto.trace.v1.Span}.
           #
-          # @return [Array<Span>] a frozen array of the finished {Span}s.
+          # @return [Array<SpanData>] a frozen array of the finished {SpanData}s.
           def finished_spans
```

Comment:
One active question here, should this be `def finished_span_datas` ?

Reply:
It's a good question. Let's leave it like this for now, and we can adjust when the SDK spec is fleshed out.